src/I18nService.js

```diff
@@ -49,6 +50,13 @@ export class I18nService extends EventEmitter {
       return this.replacePlaceholders(dictionary[key], args)
     }
 
+    if (language !== this.options.defaultLanguage && !this.fallbackToKey) {
+      console.warn(
```

Comment:
vrem acest warning in consola la fiecare cheie tradusa ? daca traduci un ecran intreg o sa fie plina consola

Reply:
Da ... vrem sa stim cand o cheie lipseste (asa era bahaviourul inainte)